config/checkstyle_checks.xml

```diff
@@ -55,6 +55,10 @@
     <property name="offCommentFormat" value="CSOFF\: .*"/>
     <property name="onCommentFormat" value="CSON\: .*"/>
   </module>
+  <module name="SuppressWithNearbyTextFilter">
+    <property name="nearbyTextPattern" value='name=\"message\"'/>
+    <property name="influenceFormat" value="1"/>
```

Comment:
please add idPattern , as we targeting to suppress only specific Check.

Reply:
Done.